astroquery/gemini/core.py

```diff
@@ -0,0 +1,484 @@
+"""
+Search functionality for the Gemini archive of observations.
+
+For questions, contact ooberdorf@gemini.edu
+"""
+
+from datetime import date
+
+import warnings
+
+import astropy
+from astropy import units
+from astropy.table import Table, MaskedColumn
+from astropy.utils.exceptions import AstropyUserWarning
+from ..utils import commons
+import numpy as np
+
+import json
+
+from ..query import BaseQuery
+from ..utils.class_or_instance import class_or_instance
+from ..utils import async_to_sync
+from . import conf
+
+__all__ = ['Observations', 'ObservationsClass']  # specifies what to import
+
+
+__valid_instruments__ = [
```

Comment:
note to self: consider making these visible sets that users can use as arguments (i.e. instruments.NIRI)

Reply:
I've decided I'm nervous about defining these options more concretely than the free form strings.  The danger is we see additional options supported server-side and the astroquery data structures will be behind what is actually live.  Tabling for now, but will revisit.